apps/Dataman/src/screens/branchReceive.tsx

```diff
@@ -0,0 +1,81 @@
+import { ScrollView, View } from "react-native";
+import React from "react";
+import NavBar from "../components/NavBar";
+import { Button, List, Text } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+const BranchReceive = () => {
+  return (
+    <View>
+      <NavBar />
+      <ScrollView>
```

Comment:
refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view